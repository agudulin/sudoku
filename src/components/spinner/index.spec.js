/* global describe, it, expect */
import React from 'react'
import renderer from 'react-test-renderer'

import Spinner from './'

describe('Spinner', () => {
  it('renders correctly when loading', () => {
    const tree = renderer.create(
      <Spinner loading />
    ).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
