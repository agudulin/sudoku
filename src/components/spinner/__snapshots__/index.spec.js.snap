exports[`Spinner renders correctly when loading 1`] = `
<div
  className="spinner spinner--active">
  Loadingâ€¦
</div>
`;
